packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -12,34 +12,9 @@
 		outline: 1px solid transparent;
 	}
 
-	textarea.block-editor-default-block-appender__content { // Needs specificity in order to override input field styles from WP-admin styles.
```

Comment:
Check out how much CSS we can remove due to this change. Very cool!

Reply:
Nice!